.travis.yml

```diff
@@ -15,11 +14,6 @@ env:
   - "TEST_SUITE=replication TEST_ROOT=vmdb"
   - "TEST_SUITE=automation  TEST_ROOT=vmdb"
   - "TEST_SUITE=javascript  TEST_ROOT=vmdb"
-matrix:
-  allow_failures:
-  - rvm: "2.1"
-    env: "TEST_SUITE=vmdb        TEST_ROOT=vmdb"
-  fast_finish: true
```

Comment:
Let's keep fast_finish in there.  It's nice to get immediate feedback when there's a failure.